mysite/settings.py

```diff
@@ -32,6 +32,7 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'coverage',
```

Comment:
What is this app? Where does it come from?

Reply:
I accidentally forgot about the link you shared for coverage.py and ended up following this one: https://realpython.com/testing-in-django-part-1-best-practices-and-examples/

Reply:
That's weird. It says to add it but doesn't show it as a code example. I wasn't able to find any examples of doing so on the internet. Does it work? Does anything change if you remove it?

Reply:
I've checked it, and it seems like it really doesn't affect anything. I've made changes https://github.com/kpi-web-guild/django-girls-blog-Martolivna/pull/21/commits/f5f6ceeeeae84a95111d275b0669193609030367 .

Reply:
Marking this as resolved.